examples/plot.py

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Please add the license header.

I guess you intend to make this an executable file, i.e., run this file to generate some results. So you might want to put the code in a main method, and run the main method. In that way, if this file is called directly, it will run the results; and if it is imported (for example, access the global variables defined in this file like NUM_PUBS), the code will not be run. An example:
https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/examples/basic_comparison.py

In the above example, all the executable code is included in a main function (def main). And if the file is called directly (i.e., on line 147, if __name__ == '__main__'), the code in the main function will run. Otherwise, if you import that module, the code will not be run unless you explicitly call that that main function.

Reply:
thanks! this is done now.